x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/execution/search/extractor/FieldHitExtractor.java

```diff
@@ -39,7 +40,7 @@
      */
     private static String[] sourcePath(String name, boolean useDocValue, String hitName) {
         return useDocValue ? Strings.EMPTY_ARRAY : Strings
-                .tokenizeToStringArray(hitName == null ? name : name.substring(hitName.length() + 1), ".");
+            .tokenizeToStringArray(hitName == null ? name : name.substring(hitName.length() + 1), ".");
```

Comment:
There's definitely some formatting differences here.
Anyway, make sure you set your IDE to format only the lines that were modified as that prevents unnecessary changes like the above.